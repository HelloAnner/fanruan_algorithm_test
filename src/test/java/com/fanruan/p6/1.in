10
600
1000
-1