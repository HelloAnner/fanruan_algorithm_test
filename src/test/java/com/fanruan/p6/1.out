55
110433070572952242346432246767718285942590237357555606380008891875277701705731473925618404421867819924194229142447517901959200
43466557686937456435688527675040625802564660517371780402481729089536555417949051890403879840079255169295922593080322634775209689623239873322471161642996440906533187938298969649928516003704476137795166849228875